lib/OpenQA/WebAPI/Auth/OpenID.pm

```diff
@@ -9,19 +9,19 @@ use LWP::UserAgent;
 use Net::OpenID::Consumer;
 use MIME::Base64 qw(encode_base64url decode_base64url);
 
-sub auth_login ($self) {
-    my $url = $self->app->config->{global}->{base_url} || $self->req->url->base->to_string;
+sub auth_login ($c) {
```

Comment:
We actually normally call the controller `$self` in functions of the controller packages. However, calling it `$c` is also fine I suppose.